pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -906,3 +915,50 @@ type NomadSDConfig struct {
 	// +optional
 	EnableHTTP2 *bool `json:"enableHTTP2,omitempty"`
 }
+type DockerswarmSDConfig struct {
+	// Address of the docker daemon
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	Host string `json:"host"`
```

Comment:
Reading the Prometheus code, it seems to require a URL value?

Reply:
I see that too. Could you suggest a regex pattern vaidation? 
My concern was that I was unsure of a pattern to cover all cases.

Reply:
Not sure, but I think there was an issue involving url validation at the CRD level some time back involving @slashpai. I can't recall what the final decision was.